.storybook/components/DesignTokens/Tier2/TypographyUsage.js

```diff
@@ -6,15 +6,18 @@ import { PRESET_SIZE_MAP } from '../Tier1/TypographyPresets';
 
 export class Tier2TypographyUsage extends Component {
   render() {
-    const renderTypeToken = (size, preset) => {
+    const renderTypeToken = (size, preset, legacySize) => {
       // remove variant suffix
       const normalizedPreset = preset.split('-')[0];
       const { fontSize, lineHeight } = PRESET_SIZE_MAP[normalizedPreset];
 
+      const name =
+        `eds-theme-typography-${size}` + (legacySize ? `/ ${legacySize}` : '');
```

Comment:
super nit, adding a space for readability:
```suggestion
        `eds-theme-typography-${size}` + (legacySize ? ` / ${legacySize}` : '');
```

Reply:
Updated, thanks!